miden/tests/integration/operations/stack_ops.rs

```diff
@@ -63,8 +63,10 @@ fn dupn_fail() {
         test,
         "invalid immediate: value must be in the range 0..16 (exclusive)",
         regex!(r#",-\[test[\d]+:[\d]+:[\d]+\]"#),
-        "1 | begin dup.16 end",
-        "  :           ^^",
+        "11 |",
+        "12 | begin dup.16 exec.truncate_stack end",
+        "   :           ^^",
+        "13 |",
```

Comment:
Similar to one of the other comments: why does this change now?

Reply:
Do you mean why does it display 12th line instead of 1st? 
The `truncate_stack` procedure is still located above the `begin` block for every test generated by `build_op_test`, moving lower the actual test code.

Reply:
I was more thinking about why it includes line 13 where previously it didn't.